src/main/java/net/openhft/chronicle/wire/Wires.java

```diff
@@ -522,9 +522,16 @@ public static int removeMaskedTidFromHeader(final int header) {
 
     @Nullable
     public static <E> E object0(ValueIn in, @Nullable E using, @Nullable Class clazz) {
-        Object o = in.typePrefixOrObject(clazz);
-        if (o != null && !(o instanceof Class)) {
-            return (E) in.marshallable(o, MARSHALLABLE);
+        Object o;
+        try {
+            o = in.typePrefixOrObject(clazz);
+            if (o != null && !(o instanceof Class)) {
+                return (E) in.marshallable(o, MARSHALLABLE);
+            }
+        } catch (MissingClassOrClassAliasException e) {
+            // If the type is missing, warn and try and try to read anyway
+            Jvm.warn().on(Wires.class, e.getMessage());
+            o = null;
```

Comment:
In this specific case we need to continue trying to parse it, but at least now we've logged a warning. Continuing to attempt to parse means if e.g. the SerializationStrategy returns a value from newInstanceOrNull we might have a chance at deserializing what follows.

Also we need to keep trying in order to satisfy that an unknown enum reference is parsed as a String (there's a test for that)